ceph_deploy/mon.py

```diff
@@ -232,11 +232,9 @@ def mon_add(args):
 
 
 def mon_create(args):
-
     cfg = conf.ceph.load(args)
     if not args.mon:
-        mon_initial_members = cfg.safe_get('global', 'mon_initial_members')
-        args.mon = re.split(r'[,\s]+', mon_initial_members)
+        args.mon = get_mon_initial_members(args)
 
```

Comment:
@trhoden 
I save the cfg variable because the catch_mon_errors() would use it in later line _**276**_ .
This will make us to call conf load twice in the mon_create().
one in the first, another in the get_mon_initial_members().

If we pass the _args_ to catch_mon_errors(), can we do the config load in catch_mon_error()?